Context/WebContext.php

```diff
@@ -167,6 +167,10 @@ public function generate($container)
             $js = $this->addPreviewCode($baseJs, $hasPreviewRelease, $isPreviewRelease, $container);
             $js = str_replace(array('/*!! initContainerHook */', '/*!!! initContainerHook */'), $initContainer, $js);
 
+            $ignoreGtmDataLayer = isset($container['ignoreGtmDataLayer']) && $container['ignoreGtmDataLayer'] === 1 ? 'true' : 'false';
```

Comment:
@snake14 This `===` created a problem for me while testing as the value was being returned as string not int
![Screenshot from 2023-11-08 07-29-34](https://github.com/matomo-org/tag-manager/assets/14902277/827037fa-d473-40f7-ae9f-1da684ecd6e2)

Reply:
Interesting. I was testing with PHP 8.1 which provides an Integer. Are you testing with an older version of PHP?

Reply:
Any objections to me changing it to a double equals (`==`)? I usually try to avoid doing that, but that seems like the easiest option.

Reply:
No objections, good to change it to `==`